llvm/test/Transforms/FatLTOCleanup/basic.ll

```diff
@@ -0,0 +1,46 @@
+
+; RUN: opt -passes="fatlto-cleanup" -mtriple=x86_64-unknown-fuchsia < %s -S | FileCheck %s
```

Comment:
Use UTC please. Also, is the triple here really needed?

Reply:
It's not. I've removed it.